test/helper.rb

```diff
@@ -51,6 +51,10 @@ def strategy
       end
     end
   end
+
+  def http_site
+    strategy.client.site.sub(/^https?/, 'http')
+  end
```

Comment:
Sorry, don't understands. Is it about https://github.com/mkdynamic/omniauth-facebook/pull/245#discussion_r68485495 comment? If I remove this helper this expression will be duplicated 4 times in https://github.com/mkdynamic/omniauth-facebook/pull/245/files#diff-2ae320f27491ec0176a6f06ea4c44cedR102